docs/known_issues.rst

```diff
@@ -388,3 +388,37 @@ this can be fixed by updating to the latest version of the ``_astropy_init.py``
 file. The root cause of this issue is that pytest now tries to pick up the
 top-level ``test()`` function as a test, so we need to make sure that we set a
 ``test.__test__`` attribute on the function to ``False``.
+
+
+Workarounds for Unspported Versions
+===================================
+
+These are problems in *earlier* versions of Astropy which are caused by
+changes to something outside of the control of Astropy itself.  Workarounds
+are provided as a way to potentially keep code that depends on older versions
+functioning.  This are to be considered "as-is" or "best effort", though, as
+older versions of Astropy cannot be supported for arbitrarily long amounts of
+time.
+
+IERS server changes
+-------------------
+
+In late 2019, the canonical server for providing the most precise earth rotation
+parameters (the USNO IERS server) was taken offline for an extended period.
+While current versions of Astropy have been updated to use a mirror server,
+older versions (i.e. prior to 3.2.3) still have as their default the server
+that is unavilable (at the time of this writing).  A workaround is to use the
+following code::
+
+    from astropy.utils import iers
+    iers.Conf.iers_auto_url.set('ftp://cddis.gsfc.nasa.gov/pub/products/iers/finals2000A.all')
+
+this must be executed *before* any code that makes use of IERS (much of
+`astropy.coordinates` and significant portions of `astropy.time`) - the safest
+choice is to run it before *any* other Astropy code.
```

Comment:
"the safest choice" is kind of scary catch-all language.  Can we be more specific here?  e.g., "This code must be executed before any other code that uses.... If `coordinates` or `time` features are used prior to invoking the above lines, the original URL will be cached, and the cache cannot (easily?) be cleared."

Or, if the cache can be cleared, link to those docs.  

I don't mean to draw this out, as this text is already helpful, but "safest" suggests other options are unsafe, and frankly, I want to know what the underlying operations are... I think you could also do something like clear cache, then reset the URL, then start doing stuff?